cypress/support/commands/api-commands.js

```diff
@@ -31,9 +31,13 @@ Cypress.Commands.add('authenticate', () => {
 
             cy.log('request /api/oauth/token')
 
-            // cookie should expire before the token is expired
-            return cy.setCookie('_apiAuth', JSON.stringify(result), { expiry: result.expiry - 10})
-                .then(() => result)
+            // auth token needs to have at least x seconds of lifetime left
+            const minimumLifetime = Cypress.env('minAuthTokenLifetime') || 60;
```

Comment:
Shouldn't we add those 60 seconds to `cypress.json` directly?

Reply:
We can do that in platform, sure

Reply:
Sounds like a plan :)

Reply:
I think we should leave the default there, so that other users of the library don't have to define this env var.

Reply:
Yeah you're right about that